.ci/bl-specific.sh

```diff
@@ -1,6 +1,14 @@
 #!/bin/bash
 
-# cp -v <...> ~/.ipython/profile_${TEST_PROFILE}/
+export AZURE_TESTING=1
+
+sudo apt-get update && sudo apt-get install redis
+
+sudo systemctl status redis
+sudo systemctl start redis
+sudo systemctl status redis
+
+if [ "$CONDA_ENV_NAME" == "collection-2021-1.0" ]; then
+    conda remove wxpython --force -y
+fi
```

Comment:
**A note to ourselves:** the currently used conda environment deployed to the floor (`collection-2021-1.0`) has an issue with importing `larch`, which subsequently imports `wx` (provided by the `wxpython` package). The latter one was recently removed from the list of `xraylarch`'s dependencies via https://github.com/nsls-ii-forge/xraylarch-feedstock/pull/9, which is reflected in the newer conda environment `collection-2021-1.2` (not deployed yet). We are removing the `wxpython` dependency in `collection-2021-1.0` to allow for successful import of `larch`.